diff --git a/node_modules/svelte/compiler.js b/node_modules/svelte/compiler.js
index a3abbbf..fcdb069 100644
--- a/node_modules/svelte/compiler.js
+++ b/node_modules/svelte/compiler.js
@@ -16621,12 +16621,14 @@
 	            parser.meta_tags[name] = true;
 	        }
 	    }
+		// FIXME: slots are not parsed as Slot AST entry
 	    const type = meta_tags.has(name)
 	        ? meta_tags.get(name)
 	        : (/[A-Z]/.test(name[0]) || name === 'svelte:self' || name === 'svelte:component') ? 'InlineComponent'
 	            : name === 'svelte:fragment' ? 'SlotTemplate'
 	                : name === 'title' && parent_is_head(parser.stack) ? 'Title'
-	                    : name === 'slot' && !parser.customElement ? 'Slot' : 'Element';
+	                    // : name === 'slot' && !parser.customElement ? 'Slot' : 'Element';
+	                    : name === 'slot' ? 'Slot' : 'Element';
 	    const element = {
 	        start,
 	        end: null,
